database/models/ussds.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+module.exports = (sequelize, DataTypes) => {
+  const Ussds = sequelize.define(
+    'Ussds',
+    {
+      sessionId: DataTypes.STRING,
+      serviceCode: DataTypes.STRING,
+      level: DataTypes.INTEGER,
+      fname: DataTypes.STRING,
+      lname: DataTypes.STRING,
+      county: DataTypes.STRING,
+      scounty: DataTypes.STRING,
+      ward: DataTypes.STRING,
+      ftype: DataTypes.STRING,
+      gender: DataTypes.STRING,
+    },
+    {}
+  );
+  Ussds.associate = function (models) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['models' is defined but never used.](https://app.codacy.com/gh/BuildForSDG/Farm-Pro/pullRequest?prid=5674718)